user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Query_Data_Sources/Get_views.md

```diff
@@ -5,3 +5,10 @@ uid: Get_views
 # Get views
 
 Available from DataMiner 10.2.0/10.1.4 onwards. The *Get views* data source retrieves a list of the views in the DMS. By default, only the columns *View ID* and *Name* are included, but you can include additional columns using a [*Select* operator](xref:GQI_Select).
+
+## Updates
+
+From DataMiner 10.3.11 onwards, this data source supports [real-time updates](xref:Query_updates).
+
+> [!WARNING]
+> There are no updates yet for views that are deleted. That means rows for deleted views will not be removed automatically until you re-execute the query.
```

Comment:
Do we need to specify the support for additions?
Do we need to address the support of meta data?

Reply:
I think it's ok to only specifically mention the things that don't work yet but you would expect to work?
I'll add a note for the metadata.

Reply:
Are additions on views already supported?

Reply:
Yes those worked as expected. It's only for deletions that we didn't get any SLNet events.
See task: [https://collaboration.dataminer.services/task/220031](https://collaboration.dataminer.services/task/220031)